src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = days * 40;
+
+  if (days >= 7) {
+    return rentalCost - 50;
+  }
+
+  if (days >= 3) {
+    return rentalCost - 20;
+  }
+
+  return rentalCost;
```

Comment:
The function name 'calculateRentalCost' is descriptive enough, but the variables and literals inside the function should be replaced with named constants to improve readability and maintainability. For example, the daily rental cost, the long term rental threshold, and the discount amounts should be defined as constants at the beginning of the function.